config/env.js

```diff
@@ -20,5 +21,6 @@ module.exports = Object
     env['process.env.' + key] = JSON.stringify(process.env[key]);
     return env;
   }, {
-    'process.env.NODE_ENV': NODE_ENV
+    'process.env.NODE_ENV': NODE_ENV,
+    'process.env.NODE_PATH': NODE_PATH
```

Comment:
`env.js` is only used for variables *injected into the app*.
Doesn't seem like it's useful to expose it to the path.